Terminal.Gui/Drawing/ColorScheme.cs

```diff
@@ -215,7 +215,7 @@ private class SchemeNameComparerIgnoreCase : IEqualityComparer<string>
     {
         public bool Equals (string x, string y)
         {
-            if (x != null && y != null)
+            if (x is { } && y is { })
```

Comment:
Can we get rid of the two warnings in this file while here?

Reply:
Sure thing. I have a few additional minor code quality changes as well, taking care of other analyzer suggestions.

Reply:
That class was removed. It was unused, has a built-in equivalent, and is private, so was just unneeded code.